molgenis-data-migrate/src/main/java/org/molgenis/data/version/v1_9/Step15AddDefaultValue.java

```diff
@@ -0,0 +1,139 @@
+package org.molgenis.data.version.v1_9;
+
+import static java.util.stream.StreamSupport.stream;
+
+import javax.sql.DataSource;
+
+import org.molgenis.data.MolgenisDataException;
+import org.molgenis.data.Repository;
+import org.molgenis.data.RepositoryCollection;
+import org.molgenis.data.elasticsearch.SearchService;
+import org.molgenis.data.meta.AttributeMetaDataMetaData;
+import org.molgenis.data.meta.MetaDataService;
+import org.molgenis.data.meta.MetaDataServiceImpl;
+import org.molgenis.data.mysql.AsyncJdbcTemplate;
+import org.molgenis.data.mysql.MysqlRepository;
+import org.molgenis.data.mysql.MysqlRepositoryCollection;
+import org.molgenis.data.support.DataServiceImpl;
+import org.molgenis.data.version.MolgenisUpgrade;
+import org.molgenis.fieldtypes.StringField;
+import org.molgenis.model.MolgenisModelException;
+import org.molgenis.security.core.runas.RunAsSystemProxy;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.jdbc.core.JdbcTemplate;
+
+import com.google.common.base.Preconditions;
+
+/**
+ * Migration for the introduction of attributes attribute "defaultValue".
+ * <ol>
+ * <li>Creates the column in the database</li>
+ * <li>Iterates over the already imported JPA entities to write their defaultValue to the newly created column</li>
+ * <li>Sets the defaultValue for the generateToken attribute of the Script entity</li>
+ * <li>Recreates the attributes index in elasticsearch to reflect these changes</li>
+ * </ol>
+ * 
+ * @author fkelpin
+ */
+public class Step15AddDefaultValue extends MolgenisUpgrade
+{
+	private static final Logger LOG = LoggerFactory.getLogger(Step15AddDefaultValue.class);
+	private final JdbcTemplate jdbcTemplate;
+	private final DataSource dataSource;
+	private final SearchService searchService;
+	private final RepositoryCollection jpaRepositoryCollection;
+
+	public Step15AddDefaultValue(DataSource dataSource, SearchService searchService,
+			RepositoryCollection jpaRepositoryCollection)
+	{
+		super(14, 15);
+		this.dataSource = Preconditions.checkNotNull(dataSource);
+		this.jdbcTemplate = new JdbcTemplate(dataSource);
+		this.searchService = Preconditions.checkNotNull(searchService);
+		this.jpaRepositoryCollection = Preconditions.checkNotNull(jpaRepositoryCollection);
+	}
+
+	private String createUpgradeSql(String entityName, String attributeName, String defaultValue)
+	{
+		return String.format("UPDATE attributes SET defaultValue = '%s' WHERE name = '%s' "
+				+ "AND identifier IN (SELECT attributes FROM entities_attributes ea WHERE ea.fullName = '%s');",
+				defaultValue, attributeName, entityName);
+	}
+
+	@Override
+	public void upgrade()
+	{
+		LOG.info("Updating metadata from version 14 to 15");
```

Comment:
I understand from the comments that the number is discussed, however it will now break continues integration on molgenis01 if merged right away. Step 15 will be executed before the not yet merged 14, so step 14 will never execute when merged.